praw/decorators.py

```diff
@@ -48,7 +48,14 @@ def alias_function(function, class_name):
     """
     @wraps(function)
     def wrapped(self, *args, **kwargs):
-        func_args = inspect.getargspec(function).args
+        if six.PY3 and not hasattr(sys, 'pypy_version_info'):
```

Comment:
It could be another nested function within `alias_function` since it doesn't really fit anywhere else at the moment. If we used it elsewhere, I think internal would make sense.